nixos/modules/services/security/sks.nix

```diff
@@ -99,6 +116,11 @@ in {
           ${lib.optionalString (cfg.webroot != null)
             "ln -sfT \"${cfg.webroot}\" web"}
           mkdir -p dump
+          ${lib.optionalString (cfg.extraDbConfig != "")
+            ''
+              ln -sfT ${dbConfig} KDB/DB_CONFIG
+              ln -sfT ${dbConfig} PTree/DB_CONFIG
+            ''}
```

Comment:
This is not quite correct, as going from some extradbconfig to an empty one will cause the old (stale/wrong) config to persist.

Reply:
What this does is that if you don't set the extraDbConfig option, it changes nothing at all. And if you set it you'll get a symlink in each place to the config you defined.

This was to not break existing installs.

But maybe it's worth setting something by default, like all the time?

Reply:
Yes, probably so. The problem wit the PR as-is is you currently can't go from a config to no config.

Reply:
So you think it's better to not have it as an optional so it always overwrites the files? That way it would go to have an empty config. Hence it would go from having a config file manually created to a nix-managed empty one.